app/code/Magento/ReleaseNotification/Ui/DataProvider/NotificationDataProvider.php

```diff
@@ -82,7 +82,9 @@ public function __construct(
     }
 
     /**
-     * {@inheritdoc}
+     * Get data
+     *
+     * @return mixed
```

Comment:
Please, revert changes related to @inheritdoc. Replace {@inheritdoc} with @inheritdoc